src/test/java/org/openapitools/jackson/nullable/JsonNullableValueExtractorTest.java

```diff
@@ -17,15 +19,23 @@
 
 
 public class JsonNullableValueExtractorTest {
-
+    private ValidatorFactory factory;
     private Validator validator;
 
     @Before
     public void setUp() {
-        ValidatorFactory factory = Validation.buildDefaultValidatorFactory();
+        factory = Validation.byDefaultProvider()
+                .configure()
+                .messageInterpolator(new ParameterMessageInterpolator())
```

Comment:
When manually configuring the `messageInterpolator`, the dependency on `javax.el` can be omitted